src/main/java/org/cloudbees/literate/jenkins/ParametersDefinitionBranchProperty.java

```diff
@@ -25,56 +25,22 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Extension;
-import hudson.model.JobProperty;
-import hudson.model.ParameterDefinition;
-import hudson.model.ParametersDefinitionProperty;
+import hudson.model.Job;
+import hudson.model.Run;
 import jenkins.branch.BranchPropertyDescriptor;
 import jenkins.branch.MultiBranchProjectDescriptor;
-import jenkins.branch.ProjectDecorator;
+import jenkins.branch.ParameterDefinitionBranchProperty;
 import org.kohsuke.stapler.DataBoundConstructor;
-import org.kohsuke.stapler.export.Exported;
-
-import java.util.Iterator;
-import java.util.List;
 
 /**
  * Support for build parameters.
  */
-public class ParametersDefinitionBranchProperty extends LiterateBranchProperty {
-
-    private final List<ParameterDefinition> parameterDefinitions;
+public class ParametersDefinitionBranchProperty extends ParameterDefinitionBranchProperty {
```

Comment:
The original name did not follow the model of `ParameterDefinition`, and for settings compatibility reasons cannot be changed now.